core/controllers/admin.py

```diff
@@ -1782,8 +1782,8 @@ def delete(self) -> None:
         Raises:
             UnauthorizedUserException. Only the default system admin can
                 manage super admins.
-            InvalidInputException. No such user exists.
-            InvalidInputException. Cannot revoke privileges from the default
+            PageNotFoundException. No such user exists.
+            UnauthorizedUserException. Cannot revoke privileges from the default
```

Comment:
Looking at https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403, this is probably more of a 403 Forbidden than a 401 Unauthorized?

I'm fine with making a new error class for that, or alternatively leaving it as-is since it's a bit outside the scope of this PR. I think it's kind of nice to make it a 403 since when analyzing server errors we can focus more on the 400s and 500s, but your call.